### DISCUSSION TITLE
数据库不全

### DISCUSSION THREAD

**Discussion by desireand** [Yin-Hongwei/vue-spring-music#202] [General]
哥，你这是不是缺了一张banner表
![image](https://user-images.githubusercontent.com/93909979/184788868-f44a6981-b5f9-41ce-b845-39413325681b.png)

**Comment by zzzzzzzzyt** [comment]
急得话可以去拉下我的仓库，我之前改了忘记放sql了，原创老哥上线了应该会合我的代码
